### PR TITLE
Please update 

### PR DESCRIPTION
**Pull request by Mridulbirla13**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request

- A GCI 2017 task
- Added my profile information in students.yml
- Added my image in img/students section.
- Here is my live link: https://codein.withgoogle.com/dashboard/task-instances/5813642547167232/

### CODE REVIEW

**Review by isuruAb — COMMENTED** [review]
Hello @Mridulbirla13 , could you please squash these two commits.
Follow this tutorial to squash two commits
https://www.youtube.com/watch?v=vp7-NeURmFg&t=44s
cheers 👍

**Comment by abishekvashok** [comment]
@Mridulbirla13 please upload your image at a dimension of exactly **240x240px**.

File: _data/students.yml [hunk 1]
```diff
@@ -169,3 +169,9 @@
   github: Jeewal
   image: Jeewal.jpg
   twitter: jeewalkunwar1
+  
+- name: Mridul Birla
+  country: India
+  github: Mridulbirla13
+  image: Mridul.jpg
+  twitter: @mridulbirla13
```
Comment (ParthS007) [inline _data/students.yml]: Please remove `@` from here.

Comment (ParthS007) [inline _data/students.yml:177]: Please remove `@` from here

**Review by ParthS007 — CHANGES_REQUESTED** [review]

**Comment by ParthS007** [comment]
@Mridulbirla13  Please don't open duplicate PR's

**Pull request closed without merging**